apps/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -315,13 +315,21 @@ export abstract class BaseInstallManager {
     const hookSource: string = path.join(this._rushConfiguration.commonFolder, 'git-hooks');
     const git: Git = new Git(this.rushConfiguration);
     const hookDestination: string | undefined = git.getHooksFolder();
+    const hooksSourceSameAsDestination: boolean = hookSource === hookDestination;
 
-    if (FileSystem.exists(hookSource) && hookDestination) {
+    if (hooksSourceSameAsDestination) {
+      console.log(
+        `git config core.hooksPath points to "common/git-hooks" folder, no need for separate git hooks installation`
+      );
+    }
```

Comment:
There's some cleanup going on in the copy script below, so I'm not sure whether skipping it is actually a good idea. On the other hand, making copies write over their source is probably bad as well.